### ISSUE TITLE
Break long urls so that content is not wider than screen

### ISSUE THREAD

**Issue by axamei** [pulibrary/orangelight#2170]
https://catalog.princeton.edu/catalog/5566741 produced a mobile usability issue via Google search console. The value for "System details" includes a long URL that should be wrapping. 

<img width="439" alt="Screen Shot 2020-07-21 at 5 20 32 PM" src="https://user-images.githubusercontent.com/8161144/88108364-b5820d00-cb76-11ea-991d-61a6fbd49e6f.png">

Add the following rule to the `dd` selector in the main-content:
```
word-break: break-word;
```
